docs/underscore.collections.statistics.md

```diff
@@ -0,0 +1,151 @@
+### statistics
+
+> Statisctics Functions. <a href="docs/underscore.collections.statistics.js.html" class="btn btn-primary btn-xs">View Annotated Source</a>
+
+#### mean
+
+Signature: `_.mean(... arrays:Array ...)`
+
+The `_.mean` function finds out the average value from the array of numbers.
+
+```javascript
+
+_.mean([]);
+//=> 0
+
+_.mean([0, 1, 2, 3, 4]);
+//=> 2
+
+_.mean(null)
+//=> 0
+
+```
+
+#### median
+
+Signature: `_.median(... arrays:Array ...)`
+
+The `_.median` function finds out the middle value from the array of numbers.
+
+Calulation of median is done using the following method.
+
+If the array has odd numbers then median is the middle element.
+
+If the array has even numbers then average of middle two numbers is the median value.
```

Comment:
Odd/even *number of* numbers. For brevity and less confusion, though, I suggest "length".

```suggestion
If the array has odd length then median is the middle element.

If the array has even length then average of middle two numbers is the median value.
```